lib/recurly/BaseClient.js

```diff
@@ -118,6 +116,25 @@ class BaseClient {
     })
   }
 
+  // Converts array parameters to CSV strings to maintain consistency with
+  // how the server expects the request to be formatted while providing the
+  // developer with an array type to maintain developer happiness!
```

Comment:
I'm a little confused by this comment. The function does more than this. It encodes the query params into the url. perhaps we put the comment about formatting arrays where the code lives on line 127.